### PR TITLE
fix: MD023/heading-start-left/header-start-left

### PR DESCRIPTION
**Pull request by nschonni**
Headings must start at the beginning of the line

### CODE REVIEW

File: iis/configuration/system.applicationHost/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ ms.assetid: e809012e-b83b-41c3-bc1d-7c479e29f622
 msc.legacyurl: /configreference/system.applicationhost
 msc.type: config
 ---
- &lt;system.applicationHost&gt;
+&lt;system.applicationHost&gt;
```
Comment (mairaw) [inline iis/configuration/system.applicationHost/index.md]:
```suggestion
# &lt;system.applicationHost&gt;
```

File: iis/configuration/system.applicationHost/index.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ ms.assetid: e809012e-b83b-41c3-bc1d-7c479e29f622
...
 ====================
```
Comment (mairaw) [inline iis/configuration/system.applicationHost/index.md]:
```suggestion
```

File: iis/configuration/system.ftpServer/index.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ ms.assetid: e6d08507-d023-47f5-9359-53bfb1183953
 msc.legacyurl: /configreference/system.ftpserver
 msc.type: config
 ---
- &lt;system.ftpServer&gt;
+&lt;system.ftpServer&gt;
```
Comment (mairaw) [inline iis/configuration/system.ftpServer/index.md]:
```suggestion
# &lt;system.ftpServer&gt;
```

File: iis/configuration/system.ftpServer/index.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ ms.assetid: e6d08507-d023-47f5-9359-53bfb1183953
...
 ====================
```
Comment (mairaw) [inline iis/configuration/system.ftpServer/index.md]:
```suggestion
```

**Comment by nschonni** [comment]
I skipped that conversion before, because I saw 800+ hits for that style of H1 in the repo. Probably something that could be regex'd to bulk-replace

**Comment by nschonni** [comment]
@mairaw rebased and disabled the rule

**Pull request merged**